pkg/stack/stack_processor_utils.go

```diff
@@ -6,6 +6,7 @@ import (
 	"os"
```

Comment:
It would help to get a second look at this file since this is where the core logic of this PR is.